src/analysis/inner_product_space/gram_schmidt_ortho.lean

```diff
@@ -103,103 +109,193 @@ open submodule set order
 
 /-- `gram_schmidt` preserves span of vectors. -/
 lemma span_gram_schmidt (f : ℕ → E) (c : ℕ) :
-  span 𝕜 (gram_schmidt 𝕜 f '' Iic c) = span 𝕜 (f '' Iic c) :=
+  span 𝕜 (gram_schmidt 𝕜 f '' Iio c) = span 𝕜 (f '' Iio c) :=
```

Comment:
Independent of how to do the generalization to finitely many vectors, this change definition looks like a good one. Can you split this change into a standalone PR while the other changes are reviewed?